plugins/woocommerce-blocks/assets/js/data/payment/test/reducers.js

```diff
@@ -151,7 +151,12 @@ describe( 'paymentMethodDataReducer', () => {
 				isSuccessful: false,
 			},
 			availablePaymentMethods: {},
-			availableExpressPaymentMethods: [ 'my-new-method' ],
+			availableExpressPaymentMethods: {
+				'my-new-method': {
+					name: 'My New Method',
+					description: 'A new way to pay quickly.',
```

Comment:
Nice catch, but looking at the type def I don't think the data store supports `description` 

```ts
/**
 * Used to represent payment methods in a context where storing objects is not allowed, i.e. in data stores.
 */
export type PlainPaymentMethods = Record< string, { name: string } >;

/**
 * Used to represent payment methods in a context where storing objects is not allowed, i.e. in data stores.
 */
export type PlainExpressPaymentMethods = PlainPaymentMethods;
```

Reply:
Thank you for sharing the datatype. I have removed the description field now. 🙂